docs/userguide/operation/proxy.md

```diff
@@ -20,13 +20,13 @@ While KumoMTA will work with any compliant SOCKS5 proxy server, we have built Ku
 
 ### Configuring an egress_source for SOCKS5 Proxy Use
 
-Configuring an egress_source to use a SOCKS5 proxy server is done as part of the `define_egress_source`
+Configuring an egress_source to use a SOCKS5 proxy server is done as part of the `make_egress_source`
 function call:
 
 ```lua
 kumo.on('init', function()
   -- Make a source that will emit from 10.0.0.1, via a proxy server
-  kumo.define_egress_source {
+  kumo.make_egress_source {
```

Comment:
similarly, this example should be updated to reference the `get_egress_source` event